lib/fake_idp/application.rb

```diff
@@ -69,5 +80,39 @@ def attributes_xml(attributes)
         %[<saml:Attribute Name="#{name}">#{attribute_value}</saml:Attribute>]
       end
     end
+
+    # This method overrides its definition in the ruby-saml-idp gem
```

Comment:
I assume most of this method is the same as the original definition? If so, I would expand on this comment to elaborate on exactly where the difference is in the overridden definition.